content/api/overview/overview.md

```diff
@@ -6,4 +6,6 @@ order: 1
 ## Overview
 This section details Datadogs HTTP API. It makes it easy to get data in and out of Datadog.
 
-The Datadog API uses resource-oriented URLs, uses status codes to indicate the success or failure of requests and returns JSON from all requests. Let's dive in and see how it works.
\ No newline at end of file
+The Datadog API uses resource-oriented URLs, uses status codes to indicate the success or failure of requests and returns JSON from all requests. Let's dive in and see how it works.
+
+**Note**: cURL code examples assume usage of BASH and GNU coreutils. On OSX you can install coreutils via the Homebrew package manager: `brew install coreutils`
```

Comment:
Let's make "Homebrew package manager" a link to https://brew.sh/

Reply:
👍